adminutils/adminutils.py

```diff
@@ -187,31 +187,48 @@ async def emoji_add(self, ctx, name: str, url: str, *roles: discord.Role):
         else:
             await ctx.tick()
 
-    @emoji.command(name="message", aliases=["steal"])
-    async def emote_steal(self, ctx, name: str, message_id: discord.Message, *roles: discord.Role):
+    @emoji.command(name="steal", aliases=["member", "message"])
+    async def emoji_steal(self, ctx, id: Union[discord.Message, discord.Member, discord.Emoji, discord.PartialEmoji], name: Optional[str], *roles: discord.Role):
         """
-        Add an emoji from a specified message
+        Add an emoji from either a member's status, a message, or an emoji from another server.
+        
+        A name must be passed if you would like to lock the emoji to roles.
         Use double quotes if role name has spaces
-
+        
         Examples:
-            `[p]emoji message Example 162379234070467641`
-            `[p]emoji message RoleBased 162379234070467641 EmojiRole`
+            `[p]emoji steal 236598364265634245`
+            `[p]emoji steal @user peepoDance`
```

Comment:
```suggestion
            `[p]emoji import Username#0000 peepoDance`
```
`@name` is kinda "ambiguous" in context of discord. It can fail in example if its not an actual mention, just an text. For example better to use so named ["Discord Tag"](https://moeka.is-a-good-waifu.com/7EXzd38.png).